be/src/util/raw_container.h

```diff
@@ -67,10 +67,13 @@ class RawAllocator : public A {
     static const size_t _trailing = trailing;
 };
 
+#if _GLIBCXX_USE_CXX11_ABI
 using RawString = std::basic_string<char, std::char_traits<char>, RawAllocator<char, 0>>;
-
 using RawStringPad16 = std::basic_string<char, std::char_traits<char>, RawAllocator<char, 16>>;
-
+#else
+using RawString = std::string;
```

Comment:
Please provide some comment about why disable it when _GLIBCXX_USE_CXX11_ABI is 0